getting-started-gradle/pom.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I believe we just use this pom because of CI right?
In that case, it must be clearly indicated.

Reply:
Yes it's just for CI, it delegates the build to `gradle`. Where should I indicate that? Directly in the `pom.xml` file?

Reply:
@glefloch perhaps, remove all the usual quarkus plugins and config from the pom to make its purpose less ambiguous?

Reply:
Ah, looks like you already did, sorry.

Reply:
Ah, looks like you already did, sorry.

Reply:
no problem @aloubyansky :), I added a comment in the file